Style reports layout

lmichalevich opened the pull request:
## Status

- [x] Ready
- [ ] In Progress

## Related Issues

https://github.com/demisto/etc/issues/38832

## Description
Styling table section & empty sections

## Snapshots (before)
![Screen Shot 2021-09-12 at 18 43 22](https://user-images.githubusercontent.com/73780437/133051660-153e60fc-93ea-4569-902d-15354aa29041.png)

## Snapshots (after)
<img width="631" alt="Screen Shot 2021-09-13 at 14 28 19" src="https://user-images.githubusercontent.com/73780437/133075881-5bf0cfcc-639c-410a-8574-895576866d4e.png">

yanyo87 commented on src/components/Layouts/ReportLayout.less:
```diff
@@ -112,14 +114,15 @@ h1 {
         &:not(.page-break-section) {
           border: 1px solid #DADBDB;
           padding: 10px 12px;
+          min-height: 140px;
```
from where is this value coming from

lmichalevich replied:
https://www.figma.com/proto/o1YarvORod1pfGvejrw2hk/objects-reports?page-id=156%3A12956&node-id=156%3A13059&viewport=643%2C588%2C0.08749071508646011&scaling=min-zoom&starting-point-node-id=173%3A13959

yanyo87 commented on src/components/Sections/SectionTable.less:
```diff
@@ -2,18 +2,15 @@
   overflow-x: hidden;
   margin-top: 0 !important;
 
-  thead {
-    th {
-      overflow: hidden;
-      text-overflow: ellipsis;
-    }
-  }
-
```
please revert this change we want to show the table headers

lmichalevich replied:
I restored the code the hides the table headers:
```
          table {
            display: none;
          }
```

yanyo87 commented on src/components/Sections/SectionTable.less:
```diff
@@ -2,18 +2,15 @@
     border-right: none !important;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    width: 80px;
```
why we added a fixed width ?

lmichalevich replied:
This acts like a minimum width (if there are a lot of items it will display only the first ones that fits)

yanyo87 reviewed (changes requested):
please add screenshots for before and after 

why did you decide to do all this changes? 

do we have a bug or something ?

lmichalevich commented:
@yanyo87 I updated the snapshots.

I think we have some kind of bugs with how we showed the indicators table.

In the main issue: ```In case of horizontal scroll, print only the columns that are visible in the web (when scroller is in the most left position)``` - this is why I added the width to the table columns

yanyo87 commented on src/components/Sections/SectionTable.less:
```diff
@@ -10,6 +11,7 @@
   }
 
   th, td {
+    padding: 3px !important;
```
why do we need this ?

lmichalevich replied:
override the default padding that the ui library adds there (it was larger and we need more space to show the content better)

yanyo87 commented on src/components/Sections/SectionTable.less:
```diff
@@ -1,6 +1,7 @@
 .section-table {
   overflow-x: hidden;
   margin-top: 0 !important;
+  height: 100%;
```
let's talk about this

yanyo87 commented on src/components/Layouts/ReportLayout.less:
```diff
@@ -128,6 +122,7 @@ h1 {
         .section-table {
           border: none;
           padding: 0;
+          min-height: 0;
```
let's talk about this as well

lmichalevich replied:
Will be removed

yanyo87 commented on src/components/Layouts/ReportLayout.less:
```diff
@@ -95,11 +94,6 @@ h1 {
-            height: inherit !important;
-          }
-        }
       }
```
why did we removed this ?

lmichalevich replied:
It is not needed. you can see from the pictures that table without items looks well (the border needs to be until the end of the section)
